ui/src/app/components/help-sidebar.spec.tsx

```diff
@@ -609,7 +609,7 @@ describe('HelpSidebar', () => {
 
     wrapper
       .find({ 'data-test-id': `Cromwell-expanded` })
-      .find({ 'data-test-id': 'Cromwell-settings-button' })
```

Comment:
I believe that the Clickable refactoring broke this test. I would like to change these tests from Enzyme to RTL, but I will plan on doing that in a follow up PR.